omero/sysadmins/unix/server-installation.txt

```diff
@@ -92,7 +93,7 @@ The following packages are required:
 +--------------+-------------+---------+
 | Python       | 2.7         | 2.6     |
 +--------------+-------------+---------+
-| PostgreSQL   | 9.4         | 9.3     |
+| PostgreSQL   | 9.4         | 9.2     |
```

Comment:
https://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/version-requirements.html#postgresql has 9.2 as dropped for 5.2